src/referencePackages/src/system.configuration.configurationmanager/8.0.0/Customizations.props

```diff
@@ -0,0 +1,8 @@
+<Project>
+
+  <ItemGroup Condition="'$(TargetFramework)' == 'netstandard2.0'">
+    <!-- Necessary for PermissionSet types but as a dangling compile-time only reference. -->
+    <ProjectReference Include="../../system.security.permissions/7.0.0/System.Security.Permissions.7.0.0.csproj" PrivateAssets="all" />
```

Comment:
Why was this defined as a project reference vs a package reference?

Reply:
Oh, that's coming from my other PR and was a copy paste error.